data/strings/strings.txt

```diff
@@ -24146,7 +24150,7 @@
 
   [isolines_activation_error_dialog]
     tags = android,ios
-    en = To activate and use the topographic layer please update or download the map of the area
+    en = To activate and use isolines, please update or download the map of the area
```

Comment:
```
Activating contour lines requires map data for the area
````

Seriously, are people trying to look at contour lines when they don't have data?  How about just grey it out without data, or don't warn?

Reply:
tweaked to' Activating contour lines requires downloading map data for this area'